README.md

```diff
@@ -295,4 +295,17 @@ $ source <(docker-app completion zsh)
 Set the docker-app completion code for zsh to autoload on startup in your ~/.zshrc
 ```sh
 source <(docker-app completion zsh)
-```
\ No newline at end of file
+```
+
+## Experimental
+
+Some commands are flagged as experimental and will remain in this state until they mature. These commands are only accessible using an experimental binary. Feel free to test these commands and give us some feedback!
+
+### How to build your own experimental binary
+
+```sh
```

Comment:
nit: strictly this `sh` means that what follows is a shell script or a shell snippet. For the sort of (shell based) example you are doing here you want `console`.

Other than that LGTM

Reply:
Oh nice, I didn't know that, thanks @ijc 👍